assembl/graphql/schema.py

```diff
@@ -233,46 +431,54 @@ def resolve_video(self, args, context, info):
         )
 
     def resolve_img_url(self, args, context, info):
+        images = [
```

Comment:
This is not yet developed. There is a task in sprint 28 to be able to able to upload an image for a thematic.